build.gradle

```diff
@@ -182,30 +182,30 @@ task starthere(type: Exec, dependsOn: [copyLocal, copyExtraLibs, copyTestLibs, c
 }
 
 task fastbuild(dependsOn: [
-    ':dbfit-java:core:check',
-    ':dbfit-java:db2:check',
-    ':dbfit-java:derby:check',
-    ':dbfit-java:hsqldb:check',
-    ':dbfit-java:mysql:check',
-    ':dbfit-java:netezza:check',
-    ':dbfit-java:oracle:check',
-    ':dbfit-java:postgres:check',
-    ':dbfit-java:sqlserver:check',
-    ':dbfit-java:teradata:check',
-    ':dbfit-java:derby:integrationTest',
-    ':dbfit-java:hsqldb:integrationTest'
+      ':dbfit-java:core:check'
```

Comment:
Leading comma is a bit unnatural for people to read. Besides - I think we rarely commit changes to this list. If its some personal customization - it's perhaps better to keep customizations in separate (optional) non-version controlled file which to load if present (since it will be personal, anyone can use own style and formatting).